ve/opencl/engine_opencl.cpp

```diff
@@ -245,7 +247,10 @@ cl::Program EngineOpenCL::getFunction(const string &source) {
                      << log
                      << "^^^^^^^^^^^^^ Log END ^^^^^^^^^^^^^" << endl << endl;
             }
-            cout << "************ SOURCE ************" << endl
+            fs::path srcfile = jitk::write_source2file(source, tmp_src_dir,
+                                                       source_filename,
+                                                       true);
```

Comment:
Either have all arguments on same line or one on each. You can't have two on the first and then one and one. 😅 

I would personally go for all on the same line in this case.

Reply:
Guess I should get a linter for C++ :laughing: